filaments/devildesign.json

```diff
@@ -463,6 +463,228 @@
                     "hex": "005387"
                 }
             ]
+        },
+        {
+            "name": "{color_name}",
+            "material": "SILK",
+            "density": 1.23,
+            "weights": [
+                {
+                    "weight": 1000,
+                    "spool_weight": 250
+                }
+            ],
+            "diameters": [
+                1.75
+            ],
+            "extruder_temp": 235,
+            "bed_temp": 60,
+            "colors": [
+                {
+                    "name": "White",
+                    "hex": "f1f0ea"
+                },
+                {
+                    "name": "Bright orange",
+                    "hex": "eb9c52"
+                },
+                {
+                    "name": "Bright pink",
+                    "hex": "d8a0a6"
+                },
+                {
+                    "name": "Red",
+                    "hex": "cb7375"
+                },
+                {
+                    "name": "Blue",
+                    "hex": "6093ac"
+                },
+                {
+                    "name": "Bright green",
+                    "hex": "b9ceac"
+                },
+                {
+                    "name": "Green",
+                    "hex": "237f52"
+                },
+                {
+                    "name": "Silver",
+                    "hex": "7f868a"
+                },
+                {
+                    "name": "Light gold",
+                    "hex": "e2a300"
+                },
+                {
+                    "name": "Gold",
+                    "hex": "ba8f4c"
+                },
+                {
+                    "name": "Copper",
+                    "hex": "794d3e"
+                },
+                {
+                    "name": "Bronze",
+                    "hex": "755847"
+                }
+            ]
+        },
+        {
+            "name": "{color_name}",
+            "material": "ABS+",
```

Comment:
I believe ABS+ is just ABS with perhaps a different density.

Reply:
Its a different filament. ABS with some additives or something like that. PLA and PLA+ are also different.

Reply:
with the new changes introduced in #21 ABS+ is allowed and should pass checks, as its indeed a different material.

Reply:
Agreed, ABS and ABS+ should be differentiated if possible for no other reason as that is how the market names them. ABS+ is ABS with additional additives (which may include pother polymers). More marketing than science here.